### PR TITLE
LVUIDF-24:Added static page rendering onto the main page. Code refact…

### PR DESCRIPTION
**Pull request by Fleeck**
Static pages now render on the main page.

To review:
- [ ] @rtriska 
- [ ] @vyshnevska 
- [x] @forecome 

To merge:
- [ ] @izagor
- [x] @forecome

### CODE REVIEW

File: app/assets/javascripts/components/pages/view/page_view.js.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    return {page: []};
+  },
+
+  componentDidMount: function() {
+    (new Bb.Models.Page({id: this.props.params.pageId})).fetch({
+      success: function(model) {
+        this.setState({page: model.toJSON()});
+      }.bind(this),
+      error: function(model, response) {
+        console.error(response);
```
Comment (forecome) [inline app/assets/javascripts/components/pages/view/page_view.js.jsx]: I think this is not needed in production. Please, remove console logs here and in other files.

File: app/assets/javascripts/components/pages/view/pages_view.js.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      }.bind(this),
+      error: function(collection, response) {
+        console.error(response);
+      }
+    });
+  },
+
+  handleClick: function(e) {
+    e.preventDefault();
+    var pageId = e.target.attributes["data-page-id"].value;
```
Comment (forecome) [inline app/assets/javascripts/components/pages/view/pages_view.js.jsx]: This variable is used only in one place. Please, make it one line without creating the variable.

File: app/assets/javascripts/routes.js.jsx [hunk 3]
```diff
@@ -25,6 +25,8 @@ var Router = (
    <Route path='/pages/new' name='new_page' handler={NewPage} />
    <Route path='/pages/:pageId' name='page' handler={Page} />
    <Route path='/pages/:pageId/edit' name='edit_page' handler={EditPage} />
+   <Route path='/pagesview/:pageId' name='view_page' handler={PageView} />
```
Comment (forecome) [inline app/assets/javascripts/routes.js.jsx]: Can the path be like `/pages/:id/view`?

**Comment by Fleeck** [comment]
@forecome refactored and reworked. Please, review once more.

**Pull request merged**